tests/requirements.txt

```diff
@@ -1,5 +1,6 @@
 docker
 elasticsearch
+netifaces
```

Comment:
this has to be pip installed in the python dockerfile under dev so that it's accessible in vent when it's deployed as an ISO.  it also needs to be added to `setup.py`